build/common.sh

```diff
@@ -55,7 +55,11 @@ readonly KUBE_BUILD_PPC64LE="${KUBE_BUILD_PPC64LE:-n}"
 # Constants
 readonly KUBE_BUILD_IMAGE_REPO=kube-build
 readonly KUBE_BUILD_IMAGE_CROSS_TAG="$(cat ${KUBE_ROOT}/build/build-image/cross/VERSION)"
-# KUBE_BUILD_DATA_CONTAINER_NAME=kube-build-data-<hash>"
+# KUBE_DATA_CONTAINER_NAME=kube-build-data-<hash>"
+
+# Increment this version to cause everyone to rebuild images, including the data
+# container.
```

Comment:
This is necessary when we do major changes to the build environment.  Things like bump the golang version or add new volumes to the data container.  We now also no longer `docker build` on every `build/run.sh` so if you need to rebuild the build container (say, you modify `rsyncd.sh`) then you'd want to increment this too.  This doesn't line up with major kubernetes versions.

I'll add some more text explaining this.